### DISCUSSION TITLE
Crashes when loading context menus for 7z files

### DISCUSSION THREAD

**Discussion by JTCozart** [valinet/ExplorerPatcher#1937] [General]
This might be specific to 7zip but using z7ip (version 23.01) and Windows 22621.2134 with EP version 22621.1992.56.3. Clicking on a large 7Z file to use the context menu to extract archive (tested with https://github.com/OWASP/SecurityShepherd/releases/download/v3.1/owaspSecurityShepherd_v3.2.3_MobileDevice.7z) explorer will crash.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1828

Please close this one.

Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"
